setup.py

```diff
@@ -6,6 +7,10 @@
 with open(Path(__file__).parent.absolute() / 'README.md', encoding='utf-8') as f:
     long_description = f.read()
 
+if "CIBUILDWHEEL" in os.environ and os.environ["CIBUILDWHEEL"] == "1":
+    CIBW_CMAKE_OPTIONS = ["-DCMAKE_INSTALL_LIBDIR=lib"]
+else:
+    CIBW_CMAKE_OPTIONS = []
```

Comment:
Perhaps it make more sense to always set `-DCMAKE_INSTALL_LIBDIR=lib`, what do you think? In which context to you think it makes sense to install in `lib64` ?

Reply:
This happens for the moment only in CentOS, and the problem is when this `lib64` folder is included in the wheel and installed in a platform that does not recognize it. I assume that if your distro is CentOS and you want to create locally the wheel and install it, the CMake on that platform will recognize `lib64`. If this is true, we can keep using the default folders on the platforms that support it, what do you think?

Reply:
I don't think that is the case but the explanation is long and I am on the cellphone, so let's merge it as it is.

Reply:
Yes it has to be confirmed. Ok, feel free to add other details, we can address any problem in another PR.